core/core.gradle

```diff
@@ -35,17 +35,17 @@ setVersion(project.ext.versionNumber)
 dependencies {
     // localGroovy (which is currently 2.4.x) is used. Groovy 2.5.x supports JsonGenerator which could be used
     // in SipJsonGenerator, but causes issues with downstream builds, so alternatives provided by Groovy 2.4.x are used.
-    compile localGroovy()
-    //compile 'org.codehaus.groovy:groovy-all:2.5.4'
+    //compile localGroovy()
+    compile 'org.codehaus.groovy:groovy-all:2.4.15'
 
     // The latest version.
-    compile "com.google.guava:guava:23.0"
+    compile 'com.google.guava:guava:23.0'
```

Comment:
I'm not sure why you changed the syntax from `"` to `'`. Groovy treats strings differently based on whether they are quoted with a single quote (' -- standard String) or double quote (" -- GString). See https://stackoverflow.com/questions/6761498/whats-the-difference-of-strings-within-single-or-double-quotes-in-groovy. I tend to prefer double quotes for dependencies because it makes it easier to insert substitutions for versions later on. For example:
```
    String guavaVersion = '24.0'
    compile "com.google.guava:guava:${guavaVersion}"
```

Is there a reason why you changed the syntax for these dependencies?